contracts/starknet/space.cairo

```diff
@@ -338,6 +371,12 @@ func propose{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr :
     # Verify that the caller is the authenticator contract.
     assert_valid_authenticator()
 
+    # Verify that the executor address is one of the whitelisted addresses
```

Comment:
could be cleaner to have an `assert_valid_executor()` function like we have for authentication

Reply:
you are right! 703e0dabfee218371ed6b2e8f48d96f629498c7f